pkg/agent/flowexporter/flowrecords/flow_records.go

```diff
@@ -26,18 +26,24 @@ import (
 )
 
 type FlowRecords struct {
-	recordsMap map[flowexporter.ConnectionKey]flowexporter.FlowRecord
+	recordsMap map[flowexporter.ConnectionKey]*flowexporter.FlowRecord
```

Comment:
Any particular reason for moving to pointer?

Reply:
Those are testing changes. Removed. Thanks!